parity/run.rs

```diff
@@ -241,6 +241,9 @@ fn execute_light_impl(cmd: RunCmd, logger: Arc<RotatingLogger>) -> Result<Runnin
 		None
 	};
 
+	// fetch service
+	let fetch = new_fetch_client(net_conf.https_proxy.as_ref())?;
```

Comment:
This and https://github.com/paritytech/parity-ethereum/pull/9303/files#diff-05d235380ad5b27b1e5ee6a84511a5e8R472 gave me a really hard time fighting borrow checkers. The only way I find to do this is to move `mut net_conf` and fetch creation based on `net_conf` a little bit earlier, before others moves occur.